django/common/analyzer/condition.py

```diff
@@ -66,6 +67,8 @@ def cast_value_type(self, data_value):
             # For dates, we parse the string
             # NOTE the input date format is fixed here
             cast_value = datetime.strptime(self.value, "%Y-%m-%d").strftime("%Y-%m-%dT%H:%M:%S")
+        elif self.relation == "IN":
+            cast_value = [type(data_value)(val) for val in self.value.split(",")]
```

Comment:
is it documented somewhere that you can add multiple values using commas ? not sure where the best place is, is there a "pyrog tips and tricks" page on notion ?

Reply:
I agree, we should start writing a user manual for Pyrog...